### PR TITLE
Return a promise on `start()`

### PR DESCRIPTION
**Pull request by Geczy**
This allows developers to see when BeePlugin has finished loading. Also makes available the `instance` variable on a completed load

### CODE REVIEW

**Comment by Geczy** [comment]
I removed eslintyarn because this is an unknown package?

**Comment by samuv** [comment]
I'm sorry @Geczy if I'm responding just now but in the last two weeks I was in vacations.

I will review the pull request as soon as possible.

File: src/bee.js [hunk 1]
```diff
@@ -26,11 +26,18 @@ const isValidAction = action => {
...
 
-const { LOAD, SAVE, SEND, PREVIEW, SAVE_AS_TEMPLATE, TOGGLE_STRUCTURE } = beeActions
+const {
+  LOAD,
+  SAVE,
+  SEND,
+  PREVIEW,
+  SAVE_AS_TEMPLATE,
+  TOGGLE_STRUCTURE,
+} = beeActions
```
Comment (samuv) [inline src/bee.js]: I like to leave all in one line when do not cross the 100 chars of length, can you change it?

File: config/webpack.config.module.js [hunk 2]
```diff
@@ -1,10 +1,11 @@
 /* eslint-disable */
 var webpack = require('webpack')
+var path = require('path')
 
 module.exports = {
   entry: ['whatwg-fetch', './src/bee.js'],
   output: {
-    path: './dist',
+    path:  path.join(process.cwd(), 'dist'),
```
Comment (samuv) [inline config/webpack.config.module.js]: It's All Good Here

**Comment by samuv** [comment]
Ok sorry for the hundred comments, for me, it's ok to merge if:

- Remove the versioning of the 'dist' folder? (if we don't have a really good "excuse" to do that)
- Change that line on "file formatting" commit
- Merge the fix of dependencies on package.json

What do you think? can you do that?

**Comment by Geczy** [comment]
Yes no problem

**Comment by samuv** [comment]
I'm closing this in favor of this # #16

**Pull request closed without merging**